jwst/extract_2d/extract_2d.py

```diff
@@ -18,7 +18,8 @@ def extract2d(input_model,
               tsgrism_extract_height=None,
               wfss_extract_half_height=None,
               extract_orders=None,
-              mmag_extract=99.):
+              mmag_extract=99.,
+              nbright=999):
```

Comment:
Should this default be here, if we have a default set above this scope?

Reply:
You're right that we have a default value set in the step class, but I was thinking of a use case where someone might call the `extract2d` function directly, without including `nbright`, in which case it would use this (different) default.